lib/utils.js

```diff
@@ -313,7 +314,7 @@ function removeUndefined(query, handleUndefined) {
           this.update(null);
           break;
         case 'throw':
-          throw new Error('Unexpected `undefined` in query');
+          throw new Error(g.t('Unexpected `undefined` in query'));
```

Comment:
@jannyHou Actually if I change this tests fail; they are checking the error message in test cases:

test failing:
```
expected Function { name: '' } to throw exception with a message matching /`undefined` in query/, but got 'Unexpected `{{undefined}}` in query'
```